src/navigation/services/buy-crypto/screens/BuyCryptoRoot.tsx

```diff
@@ -252,14 +254,20 @@ const BuyCryptoRoot: React.FC<
     dispatch(
       AppActions.showBottomNotificationModal({
         type: 'info',
-        title: 'Reminder',
-        message: `Keep in mind that once the funds are received in your ${selectedWallet?.currencyAbbreviation.toUpperCase()} wallet, to move them you will need to have enough funds in your Ethereum linked wallet ${
-          linkedWalletName ? `(${linkedWalletName}) ` : ' '
-        }to pay the ETH miner fees.`,
+        title: t('Reminder'),
+        message: t(
+          'Keep in mind that once the funds are received in your wallet, to move them you will need to have enough funds in your Ethereum linked wallet to pay the ETH miner fees.',
```

Comment:
We should add in capital letters the word that need to be replaced by {{variable}}